src/main/java/ch/njol/skript/util/PotionEffectUtils.java

```diff
@@ -341,4 +334,74 @@ public static List<PotionEffect> getEffects(ItemType itemType) {
 		return effects;
 	}
 
+	public static ClassInfo<PotionEffectType> getLegacyClassInfo() {
```

Comment:
It might be worth annotating/documenting this as Internal (see `ApiStatus.Internal` annotation)